docs/start/setup-channel.mdx

```diff
@@ -165,8 +165,73 @@ print(result)
     from qiskit_ibm_runtime import QiskitRuntimeService
 
     service = QiskitRuntimeService()
+  ```
+
+## Access IBM Quantum Systems using REST API
+
+Alternatively, you can also access IBM Quantum Systems using REST APIs, enabling you to work with quantum systems from any programming language or framework. 
+
+1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account) and save it to a file named `token`.
+
+2. Optionally obtain a temporary access token by supplying the API token obtained above. This is especially useful if you would like control over tokens, such as token invalidation. Alternatively, you can work directly with your IBM Quantum Platform API token.
+
+    ```python
+    import requests
+
+    with open('token') as file:
+      token=file.read()
+
+    url = 'https://auth.quantum-computing.ibm.com/api/users/loginWithToken'
+    input={'apiToken': token}
+    auth_response = requests.post(url, json=input)
+    auth_id=auth_response.json()['id']
+    ```
+
+3. Let's view your available backends
```

Comment:
It looks like there is still some funky indentation going on here, can you see that the text at the end of item 3 is too far to the left? same for end of item 4

<img width="913" alt="Screenshot 2024-06-20 at 2 20 36 PM" src="https://github.com/Qiskit/documentation/assets/23662430/ce46c6b3-282a-4657-b383-de13392f1ad1">